### PR TITLE
Add terraform_dash_in_resource_name rule

### PR DESCRIPTION
**Pull request by kulinacs**
An attempt to begin working on #273. Wanted to start with a small patch so I could get any needed feed back before making any more.

### CODE REVIEW

File: docs/rules/terraform_dash_in_resource_name.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Disallow dashes (-) in `resource` names without.
+
+## Example
+
+```hcl
+resource "aws_eip" "dash-name" {
+}
+
+resource "aws_eip" "no_dash_name" {
+}```
```
Comment (wata727) [inline docs/rules/terraform_dash_in_resource_name.md]: Looks like this line is breaking the markdown...
https://github.com/wata727/tflint/blob/3d63487837042640096def4b234b5b0832af2d14/docs/rules/terraform_dash_in_resource_name.md

**Comment by wata727** [comment]
Hi @kulinacs, Thank you for your contribution!
I would be happy to update the rule's README :)
https://github.com/wata727/tflint/blob/master/docs/rules/README.md

File: docs/rules/terraform_dash_in_resource_name.md (hunk 1 above)
Comment (kulinacs) [reply docs/rules/terraform_dash_in_resource_name.md]: Fixed!

**Comment by wata727** [comment]
👍

**Pull request merged**